src/ledger/LedgerTxn.cpp

```diff
@@ -25,9 +25,11 @@
 #include "xdr/Stellar-ledger-entries.h"
 #include "xdrpp/marshal.h"
 #include <Tracy.hpp>
-#include <algorithm>
 #include <soci.h>
 
+#include <algorithm>
```

Comment:
Is this necessary?

Reply:
`<numeric>` include is necessary for VS build, and I moved `<algorithm>` as well to logically group std vs non-std headers.